include/openPMD/Series.hpp

```diff
@@ -255,23 +262,82 @@ class Series : public Attributable
      */
     void flush();
 
+    /**
```

Comment:
For both functions, please add around a sentence on what it does before re-directing.
This is the direct, user-facing API call after all.